servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/SpliceFlatStreamToPackedSingle.java

```diff
@@ -291,14 +280,14 @@ public void onError(Throwable t) {
                 payloadSubscriber.onError(t);
```

Comment:
In lines 268-269, we should change message for `DuplicateSubscribeException`.

We should also add a note in javadoc that `tail` publisher has a constraint that users can subscribe to it only once. So, if they splice in memory publishers (like `Publisher.range(...)`) it's not a surprise for them to see `DuplicateSubscribeException`.